app/build.gradle

```diff
@@ -242,6 +242,7 @@ dependencies {
     betaImplementation Libs.leakcanary_no_op
     coverageImplementation Libs.leakcanary_no_op
     releaseImplementation Libs.leakcanary_no_op
+    implementation project(path: ':modules:permissionhandler')
```

Comment:
Suggest to move this definition to the beginning of `dependencies` section sitting with other project dependencies.